.github/CODEOWNERS

```diff
@@ -24,48 +24,36 @@
 # subdirectories.
 #/build/logs/      @doctocat
 
-/.circleci/        @bitaholic
-/.github/          @kjhman21
 /accounts/         @kjhman21
-/api/              @kjhman21
-/blockchain/       @kjhman21
-/build/            @bitaholic
-/client/           @KimKyungup
-/cmd/              @bitaholic
-/cmd/kscn/         @KimKyungup
+/api/              @aidan-kwon
+/blockchain/       @ehnuje @nogang @aidan-kwon @kjhman21
```

Comment:
@kjhman21 Could you add @KimKyungup to blockchain?
I would like to follow tx_list/pool/journal.

Reply:
Could you add @jeonghoanah to blockchain?